lib/rules/shorthand-property-no-redundant-values/index.mjs

```diff
@@ -112,6 +112,47 @@ function canCondenseToThreeValues(_top, right, _bottom, left) {
 	return right === left;
 }
 
+/**
+ * Split the `border-radius` value into horizontal and vertical radius arrays.
+ *
+ * @param {string} value
+ * @returns {{ horizontal: string[]; vertical: string[]; sawSlash: boolean }}
+ */
+function splitBorderRadius(value) {
+	/** @type {string[]} */
+	const horizontal = [];
+	/** @type {string[]} */
+	const vertical = [];
+	let sawSlash = false;
+
+	for (const node of valueParser(value).nodes) {
+		// Handle dividers: we only care about the slash
+		if (isValueDiv(node)) {
+			if (node.value === '/') {
+				sawSlash = true;
+				continue;
+			}
+
+			// Any other divider means we cannot safely lint
+			return { horizontal: [], vertical: [], sawSlash: false };
+		}
+
+		if (isVarFunction(node)) return { horizontal: [], vertical: [], sawSlash: false };
+
+		if (isValueWord(node) || isValueFunction(node)) {
```

Comment:
[question] Just to confirm, is it possible to improve the test coverage around this line?